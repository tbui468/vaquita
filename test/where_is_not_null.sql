create database sol;
open sol;

create table planets (id int8 key, name text, moons int8);
insert into planets (id, name) values (1, "Earth");
insert into planets (id, name, moons) values (2, "Mars", 2);
select * from planets where moons is not null;

close sol;
drop database sol;
exit;
