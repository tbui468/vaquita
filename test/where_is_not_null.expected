created database sol
opened database sol
created table planets
inserted 1 record(s) into planets
inserted 1 record(s) into planets
2, Mars, 2
closed database sol
dropped database sol
